test/cpp/api/functional.cpp

```diff
@@ -322,7 +322,7 @@ TEST_F(FunctionalTest, MaxUnpool1d) {
   ASSERT_EQ(y.ndimension(), 3);
   ASSERT_TRUE(
       torch::allclose(y, torch::tensor({{{0, 2, 0, 4, 5}}}, torch::kFloat)));
-  ASSERT_EQ(y.sizes(), torch::IntArrayRef({1, 1, 5}));
+  ASSERT_EQ(y.sizes(), std::vector<int64_t>({1, 1, 5}));
```

Comment:
one question and one comment:

Question: Do we actually need to make these a vector or can we just compare the IntArrayRef against the initializer_list and not specify the type?  That seems cleaner overall if it's possible.

Comment: I don't really buy that our test should be updated -- the goal of the tests should be to test, not serve as documentation for users.